package.json

```diff
@@ -8,7 +8,7 @@
 	},
 	"types": "./lib/index.d.ts",
 	"dependencies": {
-		"ardrive-core-js": "1.0.0",
+		"ardrive-core-js": "../ardrive-core-js_PE-461_migrate_code_from_cli",
```

Comment:
TODO: Replace

Reply:
Maybe at the integration branch (the base: `PE-461`) -> https://ardrive.atlassian.net/browse/PE-724